contrib/NeonVM/cell.c

```diff
@@ -41,29 +37,47 @@ Cell *cell_fromCell(const Cell *c)
 {
     assert(c != NULL);
 
+    uint32_t i = 0;
     Cell *x = cell_newCell();
     x->type = c->type;
     switch (c->type) {
         case Address:
             cell_copyCell(x, c);
             break;
+        case Array:
+            x->array_size = c->array_size;
+            x->array = malloc(sizeof(Cell) * c->array_size);
+            for (i = 0; i < x->array_size; i++) {
+                cell_copyCell(&x->array[i], &c->array[i]);
+            }
+            x->boolean = c->boolean;

```

Comment:
Since this is really an array, copying the boolean doesn't matter much, but maybe x->boolean = false;